lib/registerTaskWithProductionMode.js

```diff
@@ -2,7 +2,11 @@ const productionDefaults = {
   minify: true,
 };
 
+const developmentDefaults = {
+  sourcemaps: true,
+};
```

Comment:
Please move this block above the production one above.

Reply:
I will, good idea!